cloudwatch/helpers.go

```diff
@@ -72,8 +72,17 @@ func parseKeysTemplate(data map[interface{}]interface{}, keys string, w io.Write
 // from an interface{} map (expected to contain strings or more interface{} maps).
 // This runs once for every log line.
 // Used to fill in any template variables that may exist in the logStream or logGroup names.
-func parseDataMapTags(e *Event, logTags []string, t *fastTemplate, w io.Writer) (int64, error) {
+func parseDataMapTags(e *Event, logTags []string, t *fastTemplate, taskID string, metadata TaskMetadata, w io.Writer) (int64, error) {
 	return t.ExecuteFunc(w, func(w io.Writer, tag string) (int, error) {
+		switch tag {
+		case "ecs_task_id":
+			return w.Write([]byte(taskID))
+		case "ecs_cluster":
+			return w.Write([]byte(metadata.Cluster))
+		case "ecs_task_arn":
+			return w.Write([]byte(metadata.TaskARN))
```

Comment:
I wonder if we need to add one more if statement here like `if taskID != ""` to check if we are in ECS and we get the value. Or we can just add something in README file that we will only allow users to add task ID or metadata to the log group name when they run in ECS.

Reply:
I think you should add that check.

If they are not running in ECS, and they use any of these, then it should log an error. 

The readme should also note that they have to running in ECS as well.